src/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -85,7 +85,7 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.WaitCommEvent.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.WaitCommEvent.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.CreateFile.cs">
+    <Compile Condition="'$(TargetGroup)' != 'uap'" Include="$(CommonPath)\Interop\Windows\kernel32\Interop.CreateFile.cs">
```

Comment:
can yo uput the condition on an itemgroup? seems to be what is generally done

Reply:
done